vue-components/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
The `logo.png` does not appear to be used anymore after the switch to the library build, so I guess we can remove it. (But I don't know how to comment on a binary file on github)